tests/unit/core/server/testServer.js

```diff
@@ -31,13 +32,22 @@ describe('core/server/base', () => {
 
   function testClient({ stubRoutes = defaultStubRoutes } = {}) {
     function createStoreAndSagas() {
+      const sagaMiddleware = createSagaMiddleware();
       return {
-        store: createStore(combineReducers({ reduxAsyncConnect })),
-        sagaMiddleware: null,
+        store: createStore(
+          combineReducers({ reduxAsyncConnect }),
+          undefined,
```

Comment:
Yeah, I know it's not your fault 😄 

Makes sense, but just because it's a weird API call it could use a comment–even what you wrote here works for me.